docs/antora/README.adoc

```diff
@@ -0,0 +1,39 @@
+= Antora
```

Comment:
I'd rather have this as a markdown file since that's what we're using elsewhere for readme's in the repository

Reply:
done.